### DISCUSSION TITLE
Implementar rota de autenticação na API

### DISCUSSION THREAD

**Discussion by BancoCentraI** [community/community#144726] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Como a API lida com a validação de tokens JWT na autenticação?

**Comment by Bobonimo111** [comment, marked as answer]
A api vai receber o token e fazer a criptografia do mesmo, dependendo da biblioteca que estiver sendo utilizada pode ser muito facil, sendo em java node php o esquema é o mesmo um token jwt contem informação que so pode ser lida apos a criptografia com a frase secreta por parte do servidor.
